<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="CaseViewer">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>761</width>
      <height>381</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="caseSelect">
     <attribute name="title">
      <string>Select Case</string>
     </attribute>
     <widget class="QPushButton" name="startCase">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Case</string>
      </property>
     </widget>
     <widget class="QPushButton" name="finishCaseButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Finish Case</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadCaseButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>280</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Case</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter patient ID or load past case</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="patientIdEnter">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="prevPatientList">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>256</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="aboutCase">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>View Progress</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="spiralControl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Draw Spiral</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="accelControl">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <attribute name="title">
      <string>Accelerometer</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Trial Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recordAccel">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadAccel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>80</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelRecord">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel Record</string>
      </property>
     </widget>
     <widget class="QListWidget" name="accelCases">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="accel_trial_sel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>281</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="preopRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>preop</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="intraopRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>intraop</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="postopRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>postop</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="otherRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>other</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="trialNameAccel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="testRadio">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>129</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>test</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="intraopValue">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="resetBoard">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>50</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset BT Board</string>
      </property>
     </widget>
     <widget class="QLabel" name="accelDeviceUpdate">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>160</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ready for next recording.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImaging_Window">
    <property name="title">
     <string>HIFU</string>
    </property>
   </widget>
   <addaction name="menuImaging_Window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
